addons/web/static/tests/helpers/mock_server.js

```diff
@@ -1093,6 +1093,17 @@ var MockServer = Class.extend({
         }
         return true;
     },
+    /**
+     * Simulate a 'search' operation
+     *
+     * @private
+     * @param {string} model
+     * @param {Array} args
+     * @returns {Object[]} a list of records
+     */
+    _mockSearch(model, args) {
```

Comment:
Wait what, this wasn't needed before this point?

Reply:
apparently not... surprised me as well, looks like all tests had to mock this route by themselves.